modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java

```diff
@@ -133,19 +149,19 @@ private static Stream<DublinCoreCatalog> getDccsFromMp(MediaPackage mp, Workspac
               final var isXml = Objects.equals(mpe.getMimeType(), MimeType.mimeType("text", "xml"));
               return isCatalog && isForEpisode && isXml;
             })
-            .map(mpe -> DublinCoreUtil.loadDublinCore(workspace, mpe));
+            .map(mpe -> DublinCoreUtil.loadDublinCore(workspace, mpe))
+            .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Do you specifically want an `ArrayList`? Otherwise there is also `Collectors.toList`.

Reply:
We had that in other places too and talked about it already. Ehm. Yes I want an array list bc they are the fastest, but I'm sure `toList` also uses array lists. So :shrug:

Reply:
It was worth another try. :stuck_out_tongue: